evaluator/autoware_planning_evaluator/src/metrics_calculator.cpp

```diff
@@ -47,6 +47,9 @@ std::optional<Accumulator<double>> MetricsCalculator::calculate(
       return metrics::calcLateralDeviation(reference_trajectory_, traj);
     case Metric::yaw_deviation:
       return metrics::calcYawDeviation(reference_trajectory_, traj);
+    case Metric::nearest_pose_deviation:
+      return metrics::calcNearestPoseDeviation(
+        previous_trajectory_, reference_trajectory_, ego_pose_);
```

Comment:
previous_trajectory_ and reference_trajectory_ seems not same topic?

Reply:
```
      <remap from="~/input/trajectory" to="$(var input/trajectory)"/>   # previous_trajectory_
      <remap from="~/input/reference_trajectory" to="$(var input/reference_trajectory)"/>  # reference_trajectory_
```

Reply:
should compare same `~/input/trajectory` topic in dirrent cycle

Reply:
Changed to look at the same topic using `previous_trajectory` and `trajectory`!